xbmc/input/KeyboardStat.cpp

```diff
@@ -32,6 +32,8 @@
 #include "threads/SystemClock.h"
 #include "utils/log.h"
 
+#define HOLD_THRESHOLD 250
```

Comment:
in my joystick library I plan to allow durations on an individual button level, e.g.
 
&lt;start mod="longpress" duration="1000">ActivateWindow(Home)&lt;/guide>